packages/Utils/src/DTK_ConfigDefs.hpp

```diff
@@ -21,13 +21,8 @@
 
 #include <string>
 
-namespace DataTransferKit
-{
-
```

Comment:
Note that we were including `<cstdint>` inside `namespace DataTransferKit`.

Reply:
Ugggg